### PR TITLE
Feature/hpc rocky

### PR DESCRIPTION
**Pull request by rernst**
- Update python path
- Add container for kinship

### CODE REVIEW

File: Kinship/check_kinship.py
Comment (melferink) [inline Kinship/check_kinship.py:1]: This renaming breaks Utils/parse_child_from_fulltrio.py in which parse_ped is imported from check_kinship

File: Kinship/Dockerfile
Comment (melferink) [inline Kinship/Dockerfile:1]: i'm not experienced in making dockers, but code layout seems to be ok.
Make sure to validate the new versions of plink and king!

File: Kinship/check_kinship.py
Comment (rernst) [reply Kinship/check_kinship.py:1]: Added `parse_ped` to `Utils/parse_child_from_fulltrio.py`. It is not possible to import it (anymore) without creating an entire python module structure. I think these modules should be self contained anyway. In theory we could move CustomModules to NextflowModules.

**Review by melferink — APPROVED** [review]
All ok!

**Pull request merged**